scripts/gen_feeds_daily_off.sh

```diff
@@ -11,8 +11,9 @@ export PERL5LIB="../lib:${PERL5LIB}"
 ./remove_empty_products.pl
 ./gen_top_tags_per_country.pl
 #./compute_missions.pl
+
+# Generate CSV exports
```

Comment:
maybe move up also gziping csv vs gzipping rdf (less used) ?

Reply:
Good idea, done.

Reply:
As gzip is not really multithreaded, would it make sense to either move to a compression command that allows using multiple cores (such as [pigz](https://zlib.net/pigz/)) or maybe spin off multiple gzip processes into the background (eg, by using `&` beside the gzip commands)?

Files compressed with pigz are compatible with the gzip command.

Reply:
@foodietim we could, but the compression takes only 5 minutes per file max. One thing we should do is to compress to a new file and move it once compressed, in order to avoid someone downloading a broken file during those 5 minutes.